api/rosetta/errors.go

```diff
@@ -23,23 +23,25 @@ import (
 const (
 	invalidJSON = "request does not contain valid JSON-encoded body"
 
-	blockchainEmpty = "blockchain identifier has empty blockchain field"
-	networkEmpty    = "blockchain identifier has empty network field"
-	blockEmpty      = "block identifier has empty index and hash fields"
-	blockLength     = "block identifier has invalid hash field length"
-	addressEmpty    = "account identifier has empty address field"
-	addressLength   = "account identifier has invalid address field length"
-	currenciesEmpty = "currency identifier list is empty"
-	symbolEmpty     = "currency identifier has empty symbol field"
-	txEmpty         = "transaction identifier has empty hash field"
-	txLength        = "transaction identifier has invalid hash filed length"
+	blockchainEmpty  = "blockchain identifier has empty blockchain field"
+	networkEmpty     = "blockchain identifier has empty network field"
+	blockEmpty       = "block identifier has empty index and hash fields"
+	blockLength      = "block identifier has invalid hash field length"
+	addressEmpty     = "account identifier has empty address field"
+	addressLength    = "account identifier has invalid address field length"
+	currenciesEmpty  = "currency identifier list is empty"
+	symbolEmpty      = "currency identifier has empty symbol field"
+	txEmpty          = "transaction identifier has empty hash field"
+	txLength         = "transaction identifier has invalid hash filed length"
+	txInvalidOpCount = "transaction operations list has an invalid number of operations"
 
 	networkCheck      = "unable to check network"
 	blockRetrieval    = "unable to retrieve block"
 	balancesRetrieval = "unable to retrieve balances"
 	oldestRetrieval   = "unable to retrieve oldest block"
 	currentRetrieval  = "unable to retrieve current block"
 	txRetrieval       = "unable to retrieve transaction"
+	intentCreate      = "unable to determine transaction intent"
```

Comment:
Hm, you're right. I actually think the `create` is more fitting when observing the code and what's being done. But for the external parties, they have no notion of `intent` as an entity that's being created, so that's why `determine` sounded more fitting in the error message.

I'll switch it to `intentDetermine` to be consistent, even though I don't really like it.

Regarding the `could not` vs `unable to`, I also commented the same thing earlier, I think the idea was that these are not our internal errors, but external ones, so it's not a big deal if they're different.